modules/lofreq/1.0/config/default.yaml

```diff
@@ -0,0 +1,41 @@
+lcr-modules:
+    
+    lofreq:
+
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "__UPDATE__"
+            sample_bai: "__UPDATE__"
+
+        scratch_subdirectories: []
+
+        options:
+            lofreq: ""
+
+        switches:
+            # Capture sequencing panel in BED format
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} 
+            #                      {normal_id} {pair_status}
+            regions_bed:
+                _default: ""
+                # Set to "" if you have no capture data
+                capture: "-l __UPDATE__"
+
+        conda_envs:
+            lofreq: "{MODSDIR}/envs/lofreq-2.1.5.yml"
+            
+        threads:
+            lofreq: 48
+
+        mem_mb:
```

Comment:
See my comment below about resource unpacking. This would need to be changed to: 
```
resources: 
    lofreq: 
        mem_mb = 100000
        bam = 1
```